src/AspNetCoreModuleV2/CommonLib/iapplication.h

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+class IREQUEST_HANDLER;
```

Comment:
How did it work before?

Reply:
Not sure, but I get compilation errors saying the request handler cannot be found.

Reply:
Maybe ordering of includes in the stdafx.h file. I'll mess around with that.